src/main/java/org/example/MazeSolver.java

```diff
@@ -3,14 +3,13 @@
 import java.util.LinkedList;
 
 public class MazeSolver {
-
-    static int[][] maze = {
+    private static final int[][] MAZE = {
             {1, 0, 1, 2}, // El destino (2) está en la posición (0, 3)
             {1, 1, 0, 1},
             {0, 1, 1, 1}
     };
 
-    LinkedList<Position> path = new LinkedList<>();
+    private final LinkedList<Position> path = new LinkedList<>();
```

Comment:
I agree cause it should be final and private

Reply:
Fixed